.circleci/config.yml

```diff
@@ -192,8 +192,8 @@ jobs:
       - setup_bazel_config
       - yarn_install
       - run:
-          name: Build npm package
-          command: yarn bazel build //:npm_package --config=release
+          name: Build npm packages
```

Comment:
Maybe in a follow up, we should consider having this build everything with something like a `releasable_pkg` tag or something like that.

Reply:
I don't think that would be worth doing, since it's just two packages.